### PR TITLE
Make players die funnier

### PR DESCRIPTION
**Pull request by marchc1**
Players die with force when hit by bullets.

### CODE REVIEW

**Comment by TeeMeeFe** [comment]
Does this also affect NPC's?

**Comment by TeeMeeFe** [comment]
> > Does this also affect NPC's? Otherwise, i like this idea.
> 
> You edited after I made the comment but would you rather them _not_ affect NPC's? Just want to make sure I understand what you're saying

No, i actually would like them to be affected as well. (I should have used "anyway" instead of otherwise)

File: lua/acf/damage/damage_sv.lua [hunk 1]
```diff
@@ -40,6 +40,43 @@ function Damage.getBulletDamage(Bullet, Trace)
...
+--- Returns true if the damage has killed the player, false if it has not.
+function Damage.doSquishyFlingKill(Entity, Damage, HitPos, Attacker, Inflictor, Direction, Explosive)
+	local Health = Entity:Health()
+
+	if Damage > Health then
+		local SourceDamage = DamageInfo()
+		SourceDamage:SetAttacker(Attacker)
+		SourceDamage:SetInflictor(Inflictor)
+		SourceDamage:SetDamage(Damage)
+		SourceDamage:SetDamageForce(Direction * 200000000)
```
Comment (thecraftianman) [inline lua/acf/damage/damage_sv.lua:54]: I do like the body flinging, but a force multiplier of 200 million seems like a bit much?

**Review by thecraftianman — COMMENTED** [review]
Sorry for taking 2 months to properly review this lol, I kept forgetting about it. Overall it's probably fine besides the couple of changes that I mentioned.

**Pull request merged**